src/syft/lib/torchvision/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 # third party
 from packaging import version
+import torch as tr
```

Comment:
Q and NITPICK: Could we leave this as simple ```torch``` or ```th```?

Reply:
Yes, I'll make the change